external/fv3fit/fv3fit/pytorch/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .graph import GraphHyperparameters, train_graph_model, UNetGraphNetworkConfig
+from .graph import GraphHyperparameters, train_graph_model, MPGUNetGraphNetworkConfig
```

Comment:
In this PR you shouldn't be replacing the U-net configuration, you want to add message passing as an additional potential configuration.